content/projects/understanding-loops/_index.md

```diff
@@ -15,30 +15,30 @@ title: Shopping cart calculations
 
 A lot of noob programmers don't know why understanding is important. They think they can just copy code from people and that's enough to get by. BUT as soon as you get your first job you'll need to solve some serious problems on your own.
 
-The problems you'll be solving in this project are designed to be realistic. As a professional you'll likely need to do things similar to this. Please make sure you 100% understand!
+The problems you'll be solving in this project are designed to be realistic. As a professional, you'll likely need to do things similar to this. Please make sure you 100% understand!
 
 ## Here's the scenario
 
 Imagine you are working for a hot new online shopping startup. You've seen online shopping, right? Go poke around on Takealot or Amazon if you haven't.
 
-On the online shopping site, users (people who want to buy stuff) navigate around and then add items to their shopping baskets. Once they are happy with their baskets then they "checkout", pay for their goodies, and await their delivery.
+On the online shopping site, users (people who want to buy stuff) navigate around and then add items to their shopping baskets. Once they are happy with their baskets, then they "checkout", pay for their goodies, and await their delivery.
 
 ## Why data matters
 
-As any kind of techie, data will be a part of your life. You're going to need to learn how to make this stuff dance.
+As with any kind of techie, data will be a part of your life. You're going to need to learn how to make this stuff dance.
```

Comment:
techie is short for technical person (sortof). It's a noun. I am a techie. As a techie, I work with data.

Don't just hit things with the Grammarly hammer, make sure that sentences make sense.